HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,330 @@
+        const Eigen::Matrix3d rotationMatrixFromRollPitchYaw = rpy2rotmat(rotation.rollPitchYaw, rotation.convention);
+        std::cout << "Rotation Matrix:\n" << rotationMatrixFromRollPitchYaw << std::endl;
+
+        // Save Rotation Matrix to .yaml via OpenCV
+        fileStorageOut.write("RotationMatrixFromRollPitchYaw" + toString(rotation.convention),
+                             eigenToCv(rotationMatrixFromRollPitchYaw));
+
+        // Calculate error in transformation (Rotation Matrix --> Roll-Pitch-Yaw --> Rotation Matrix)
+        std::cout << "\nVerifying Rotation Matrix from Roll-Pitch-Yaw angles (" << toString(rotation.convention) << ")"
+                  << std::endl;
+        compareRotation(rotationMatrix, rotationMatrixFromRollPitchYaw);
+        compareRotationExtended(rotation.rollPitchYaw, rotationMatrixFromRollPitchYaw, rotation.convention);
+    }
+
+    // Rotation Vector to Axis-angle
+    std::cout << "\nConverting from rotation vector to axis-angle" << std::endl;
+    const Eigen::AngleAxisd axisAngle(representations.rotationVector.norm(),
+                                      representations.rotationVector.normalized());
+    std::cout << "\nAxis-angle:\n" << reorderAxisAngle(axisAngle) << std::endl;
+
+    // Axis-Angle to Quaternion
+    std::cout << "\nConverting axis-angle to quaternion" << std::endl;
+    const Eigen::Quaterniond quaternion(axisAngle);
+    std::cout << "\nQuaternion:\n" << quaternion.coeffs() << std::endl;
+
+    // Quaternion to Rotation Matrix
+    std::cout << "\nConverting quaternion to rotation matrix" << std::endl;
+    const Eigen::Matrix3d rotationMatrixFromQuaternion = quaternion.toRotationMatrix();
+    std::cout << "\nRotation Matrix:\n" << rotationMatrixFromQuaternion << std::endl;
+    fileStorageOut.write("RotationMatrixFromQuaternion", eigenToCv(rotationMatrixFromQuaternion));
+
+    // Calculate error in transformation (Rotation Matrix --> Quaternion --> Rotation Matrix)
+    std::cout << "\nVerifying Rotation Matrix from Quaternion" << std::endl;
+    compareRotation(rotationMatrix, rotationMatrixFromQuaternion);
+
+    fileStorageOut.release();
+}
+
+std::string toString(RotationConvention convention)
+{
+    switch(convention)
+    {
+        case RotationConvention::XYZ_Intrinsic: return "XYZ_Intrinsic";
+        case RotationConvention::XYZ_Extrinsic: return "XYZ_Extrinsic";
+        case RotationConvention::ZYX_Intrinsic: return "ZYX_Intrinsic";
+        case RotationConvention::ZYX_Extrinsic: return "ZYX_Extrinsic";
+        default: throw std::invalid_argument("Invalid RotationConvention");
+    }
+}
+
+Eigen::Vector4d reorderAxisAngle(const Eigen::AngleAxisd &axisAngle)
+{
+    Eigen::Vector4d reorderedAxisAngle;
+
+    for(int i = 0; i < 3; i++)
+        reorderedAxisAngle[i] = axisAngle.axis()[i];
+    reorderedAxisAngle[3] = axisAngle.angle();
+
+    return reorderedAxisAngle;
+}
+
+/** Converts Rotation Matrix to Roll-Pitch-Yaw angles in radians.
```

Comment:
This comment can be removed, as it contains no extra information compared to the function signature.

Reply:
So, an empty brief is preferred instead of some redundancy?